sdk/compiler/daml-lf-proto-encode/test/DA/Daml/LF/Proto3/EncodeTest.hs

```diff
@@ -363,10 +364,10 @@ typeInterningMaybeSyn =
       assertInterned pt
       assertInternedEnv e
       pt @?= ptinterned 1
-      iTypes V.! 0 @?= ptunit
-      iTypes V.! 1 @?= ptsyn 1 (V.singleton $ ptinterned 0)
-      iStrings V.! 0 @?= "Main"
-      iStrings V.! 1 @?= "MaybeSyn"
+      iTypes ! 0 @?= ptunit
+      iTypes ! 1 @?= ptsyn 1 (singleton $ ptinterned 0)
```

Comment:
~~Doesn't this need the same treatment as the test above?~~
Ah yes, the Syn exception means that we don't use TApp for type synonyms.